smac/runhistory/runhistory.py

```
@@ -594,6 +608,9 @@ def get_config(self, config_id: int) -> Configuration:
 
     def get_config_id(self, config: Configuration) -> int:
         """Returns the configuration id from a configuration."""
+        if config not in self._config_ids:
+            logger.warning("Requested id of unknown configuration!")
```

When could that happen and should we throw an error instead?